### DISCUSSION TITLE
./start.sh

### DISCUSSION THREAD

**Discussion by DariusSporis** [CoopNetwork/GMD-blockchain#9] [Q&A] [answered]
Hello again,

pentru ca la rularea scriptului ./start.sh am luat freeze am resetat tot OS-ul si am vrut sa pornesc din nou scriptul insa de data aceasta primesc o eroare.

"unique index or primary key violated..."

oare trebuie sa rad tot si sa o iau de la 0 din nou?

Multumesc pentru raspuns anticipat.

**Comment by DariusSporis** [comment]
Update, am sters folderul GMD si am riulat scriptul din fisierul .zip.

Insa din nou mi se blocheaza de data aceasta la partea de Loadin public keys.

Daca aveti vreo idee de ce sau ce as putea face in situatia aceasta, respectiv daca stiti motivul pentru care se blocheaza la pasul acesta va rog sa imi dati un feedback.

**Reply by georgeasiminei** [reply]
In caz ca primesti erorare "unique index or primary key violated..." poti sa opresti nodul, stergi directorul "nxt_db" si sa il repornesti.

Pasul cu "Loadin public keys" nu este blocat. Poate dura pana la 10 minute, depinde de hardware. BTW, ce hardware folosesti?

**Comment by DariusSporis** [comment]
Atasez si un screenshot cu partea unde se blocheaza
![Loading public keys](https://user-images.githubusercontent.com/92687362/137644287-bf122553-f64e-43c7-8139-f27d5945c615.jpg)

**Reply by DariusSporis** [reply]
Update,
pana la urma a mers. Doar ca dureaza aparent destul de mult pana se finalizeaza procesul.

**Comment by georgeasiminei** [comment, marked as answer]
Salutare. Cand folosesti prima oara nodul, dureaza pana la o ora sa porneasca sa downloadeze blockchainul si sa se sincronizeze cu celelalte noduri.

"unique index or primary key violated..." inseamna cel mai probabil ca s-a corupt baza de date cand ai dat restart fortat.
O poti reinitializa doar stergand continutul directorului "nxt_db" apoi repornesti nodul. Ai rabdare, dureaza pana se resincronizeaza nodul dupa ce ai sters baza de date (pana la o ora).

**Reply by DariusSporis** [reply]
Hello zuppermann, thanks fort the advice. I decided to start from scratch and deleted the GMD folder and the .zip file.

Downloaded again the zip file, unzipped it and wanted to start the start.sh script.

I received the attached error and noticed that the Coopnetwork.jar and Coopnetworkservice.jar were missing from my GMD folder

Any idea why that might be?

![GMD folder](https://user-images.githubusercontent.com/92687362/140406042-2899597a-3707-4a5f-846a-e6e700d7e91c.jpg)

Versiune Java>

![Java_version](https://user-images.githubusercontent.com/92687362/140408366-23fcaa06-ef1a-45f4-9852-6789aa27d47f.JPG)

**Reply by georgeasiminei** [reply]
> Hello zuppermann, thanks fort the advice. I decided to start from scratch and deleted the GMD folder and the .zip file.
> 
> Downloaded again the zip file, unzipped it and wanted to start the start.sh script.
> 
> I received the attached error and noticed that the Coopnetwork.jar and Coopnetworkservice.jar were missing from my GMD folder
> 
> Any idea why that might be?

Just checked and seen that build on node6 was generated incomplete. Please use this link https://node.thecoopnetwork.io:8443/gmd-node-standalone.zip to download the correct zip.
